web/libs/app-common/src/pages/AccountSettings/sections/Hotkeys.jsx

```diff
@@ -0,0 +1,628 @@
+import { useEffect, useState, useCallback } from "react";
+import { ToastType, useToast } from "@humansignal/ui";
+
+// Shadcn UI components
+import { Button } from "@humansignal/ui";
+import { Card, CardContent, CardHeader } from "@humansignal/shad/components/ui/card";
+import { Badge } from "@humansignal/shad/components/ui/badge";
+import { Skeleton } from "@humansignal/shad/components/ui/skeleton";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from "@humansignal/shad/components/ui/dialog";
```

Comment:
I believe it was agreed on that we were not going to reference components through this export, all components used across projects should be exported and referenced from "@humansignal/ui" exports. cc: @yyassi-heartex @nick-skriabin  @hlomzik